appveyor.yml

```diff
@@ -46,9 +46,9 @@ install:
   # Here, we use the fixed version of testthat. For more details, please see SPARK-22817.
   # As of devtools 2.1.0, it requires testthat higher then 2.1.1 as a dependency. SparkR test requires testthat 1.0.2.
   # Therefore, we don't use devtools but installs it directly from the archive including its dependencies.
-  - cmd: R -e "install.packages(c('crayon', 'praise', 'R6'), repos='https://cloud.r-project.org/')"
+  - cmd: R -e "install.packages(c('crayon', 'praise', 'R6', 'arrow'), repos='https://cloud.r-project.org/')"
   - cmd: R -e "install.packages('https://cloud.r-project.org/src/contrib/Archive/testthat/testthat_1.0.2.tar.gz', repos=NULL, type='source')"
-  - cmd: R -e "packageVersion('knitr'); packageVersion('rmarkdown'); packageVersion('testthat'); packageVersion('e1071'); packageVersion('survival')"
+  - cmd: R -e "packageVersion('knitr'); packageVersion('rmarkdown'); packageVersion('testthat'); packageVersion('e1071'); packageVersion('survival'); packageVersion('arrow')"
```

Comment:
should we specify the arrow version to install? just wonder about compatibility issue

Reply:
Maybe .. it will needs to install `devtools` which upgrades `testthat` ... so it's a bit tricky to install a specific version for now.

I was thinking about testing the latest in AppVeyor and old version at Jenkins anyway. should be fine.

Reply:
SG